### PR TITLE
Create labeled audio samples

### PR DESCRIPTION
**Pull request by ralatsdc**

### CODE REVIEW

File: DataScience/GpxWavLabeler.py [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+                    axs.set_xlabel("east [m]")
+                    axs.set_ylabel("north [m]")
+                    plt.show()
+
+
+"""Demonstrate GpxLabeler.
+"""
+if __name__ == "__main__":
+
+    gpx_filename = "suuntoapp-Motorsports-2022-02-22T18-09-01Z-track.gpx"
```
Comment (jbenjoseph) [inline DataScience/GpxWavLabeler.py:479]: Would it be possible to make these into cmd parameters? argc/argv is okay unless ArgumentParser is trivial

File: DataScience/GpxWavLabeler.py [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+
+# WGS84 parameters
+R_OPLUS = 6378137  # [m]
+F_INV = 298.257223563
+
+# Clip parameters
+DELTA_T_MAX = 4
+
+# Data and clip directories
+DATA_HOME = Path(
```
Comment (jbenjoseph) [inline DataScience/GpxWavLabeler.py:21]: Would it be possible to make these into cmd parameters? argc/argv is okay unless ArgumentParser is trivial

File: DataScience/GpxWavLabeler.py [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+    "/Users/raymondleclair/odrive/Google-Drive-RAL-Springbok/3-Shared/IQT/7-AI-Sonobuoy/Data"
+)
+CLIP_HOME = Path(
```
Comment (jbenjoseph) [inline DataScience/GpxWavLabeler.py:24]: Would it be possible to make these into cmd parameters? argc/argv is okay unless ArgumentParser is trivial

**Comment by jbenjoseph** [comment]
Ideally the usage would be something like
`python GpxWavLabeler.py my.gpx my.wav output_dir`

File: DataScience/GpxWavLabeler.py (hunk 2 above)
Comment (ralatsdc) [reply DataScience/GpxWavLabeler.py]: Sure.

File: DataScience/GpxWavLabeler.py (hunk 3 above)
Comment (ralatsdc) [reply DataScience/GpxWavLabeler.py]: Sure

File: DataScience/GpxWavLabeler.py (hunk 1 above)
Comment (ralatsdc) [reply DataScience/GpxWavLabeler.py:482]: Sure.

**Review by cglewis — APPROVED** [review]

**Pull request merged**